ci-operator/step-registry/baremetal/lab/ipi/conf/virtual-media/baremetal-lab-ipi-conf-virtual-media-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+echo "Creating patch file to configure redfish virtual media"
+cat > "$SHARED_DIR/redfish_patch_install_config.yaml" <<EOF
+platform:
+  baremetal:
+    provisioningNetwork: Disabled
+    hosts:
+EOF
+
+# shellcheck disable=SC2154
+for bmhost in $(yq e -o=j -I=0 '.[]' "$SHARED_DIR/hosts.yaml"); do
+  # shellcheck disable=SC1090
+  . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  cat >> "$SHARED_DIR/redfish_patch_install_config.yaml" <<EOF
+    - name: ${name}
+      role: ${name%%-[0-9]*}
+      bootMACAddress: ${mac}
+      rootDeviceHints:
+        ${root_device:+deviceName: ${root_device}}
+        ${root_dev_hctl:+hctl: ${root_dev_hctl}}
+      bmc:
+        address: ${redfish_scheme}://${bmc_address}${redfish_base_uri}
+        disableCertificateVerification: true
+        username: ${redfish_user}
+        password: ${redfish_password}
+      networkConfig:
+        interfaces:
+        - name: ${baremetal_iface}
+          type: ethernet
+          state: up
+          ipv4:
+            enabled: true
+            dhcp: true
+          ipv6:
+            enabled: true
```

Comment:
We have both ipv4 and ipv6 enabled, while the test name `baremetal-ipi-ovn-ipv4-amd-vmedia-f7` indicate it's ipv4. Is that expected ?

Reply:
yes. we decide upon required stack on networking section of install-config:
networking:
  machineNetwork:
  - cidr: ${INTERNAL_NET_CIDR}
  
INTERNAL_NET_CIDR is ipv4 cidr

Reply:
Ack